src/framework/global/internal/globalconfiguration.cpp

```diff
@@ -102,7 +102,7 @@ io::path GlobalConfiguration::userBackupPath() const
 
 io::path GlobalConfiguration::userDataPath() const
 {
-    static io::path p = QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation) + "/MuseScore";
+    static io::path p = QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation) + "/MuseScore4";
```

Comment:
How about making it `MuseScore4Development` for development builds? That was MuseScore 3's behaviour at least.

So that would probably be `QCoreApplication::applicationName()`, like in MU3.

Reply:
Great minds think alike, I was working on this already and have just pushed it